modules/lang-mustache/src/test/java/org/elasticsearch/action/search/template/SearchTemplateIT.java

```diff
@@ -16,137 +16,58 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.elasticsearch.messy.tests;
+package org.elasticsearch.action.search.template;
```

Comment:
Moving this to `SearchTemplateIT` means it'll run while the REST test cluster is running because it is in a module. We usually don't do that for subclasses `ESIntegTestCase`.